### PR TITLE
add glog configurable function and redirect aws log to segcore log

### PR DESCRIPTION
**Pull request by zhagnlu**
#25651

### CODE REVIEW

**Review by jiaoew1991 — APPROVED** [review]
/lgtm

**Comment by zhagnlu** [comment]
rerun ut

File: pkg/util/paramtable/service_param.go [hunk 1]
```diff
@@ -920,4 +921,12 @@ Leave it empty if you want to use AWS default endpoint`,
 		Export: true,
 	}
 	p.IAMEndpoint.Init(base.mgr)
+	p.LogLevel = ParamItem{
+		Key:          "minio.logLevel",
+		DefaultValue: DefaultMinioLogLevel,
+		Version:      "2.0.0",
```
Comment (czs007) [inline pkg/util/paramtable/service_param.go:927]: 2.3.0

**Pull request merged**